src/TransitionState.php

```diff
@@ -6,5 +6,5 @@
 
 interface TransitionState
 {
-    public function __toString();
+    public function getValue(): string;
```

Comment:
You can also extend from the `Stringable` interface in order to restore existing behaviour.

I don't specifically like `getValue` as the method name here. It's basically returning the name of the transition. Maybe `getKey` or `getName` are better alternatives?